### PR TITLE
Event confirmation support.

### LINKED ISSUE #18: Support For Event Confirmations Is Missing
Currently the Event entity does not contain confirmation details

### PR DESCRIPTION
**Pull request by christophermark**
Adds support for event confirmations.

Closes #18

### CODE REVIEW

File: src/main/java/com/robinpowered/sdk/model/Confirmation.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+/**
+ * Represents the check-in/confirmation of an {@link Event}.
+ */
+public class Confirmation {
+
+    public static final String MIME_TYPE = "vnd.robinpowered.confirmation.v1";
+
+    private String eventId;
+    private Integer user_id;
+    private Integer device_id;
```
Comment (brianmuse) [inline src/main/java/com/robinpowered/sdk/model/Confirmation.java]: Camel case?

File: src/main/java/com/robinpowered/sdk/model/User.java [hunk 2]
```diff
@@ -163,4 +163,16 @@ public Reference(int userId) {
...
+        private Integer deviceId;
+
+        public DeviceId(Integer deviceId) {
+            this.deviceId = deviceId;
+        }
+
+        public Integer getDeviceId() {
+            return deviceId;
+        }
+    }
```
Comment (brianmuse) [inline src/main/java/com/robinpowered/sdk/model/User.java]: Why's this exist?

File: src/main/java/com/robinpowered/sdk/model/Confirmation.java (hunk 1 above)
Comment (christophermark) [reply src/main/java/com/robinpowered/sdk/model/Confirmation.java]: My mistake! Not sure how this slipped by.

File: src/main/java/com/robinpowered/sdk/model/User.java (hunk 2 above)
Comment (christophermark) [reply src/main/java/com/robinpowered/sdk/model/User.java]: This allows us to use the `User.DeviceId` as the body for event confirmation requests.

Due to [this bug](https://github.com/robinpowered/robin-java-sdk/pull/21/files#diff-4fdbdff9f7f51b6bf5d246436ee3ac66R51) in 1.x versions of Retrofit, we need to include a non-null body in each request.

**Comment by christophermark** [comment]
Tests are passing now

File: src/main/java/com/robinpowered/sdk/model/User.java (hunk 2 above)
Comment (brianmuse) [reply src/main/java/com/robinpowered/sdk/model/User.java]: I don't quite follow... this feels awkward tho. Why not just `int`?

**Comment by christophermark** [comment]
@brianmuse all fixed

File: src/main/java/com/robinpowered/sdk/model/User.java (hunk 2 above)
Comment (christophermark) [reply src/main/java/com/robinpowered/sdk/model/User.java]: Not sure why I felt compelled to make this an entire class. Changed to an `int` now.

**Comment by brianmuse** [comment]
👍

**Pull request merged**